oleesir/store-manager
# #161186256User should be able to add products to cart

#### What does this PR do?
Implement add-to-cart template

#### Description of Task to be completed?
implement add-to-cart template

#### How should this be manually tested?
* Navigate to the UI folder and click on add-to-cart.html in your browser
* You will see a add-to-cart template

#### What are the relevant pivotal tracker stories?
[ #161186256](https://www.pivotaltracker.com/story/show/161186256)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="screen shot 2018-10-12 at 9 33 18 pm" src="https://user-images.githubusercontent.com/17679511/46892777-81889c80-ce66-11e8-8a79-7f32d19fb530.png">
